bundles/org.openhab.core.automation.module.script/src/main/java/org/openhab/core/automation/module/script/ScriptTransformationService.java

```diff
@@ -132,10 +148,10 @@ public void deactivate() {
             try {
                 CompiledScript compiledScript = scriptRecord.compiledScript;
 
-                if (compiledScript == null && scriptEngineContainer.getScriptEngine() instanceof Compilable) {
+                if (compiledScript == null
+                        && scriptEngineContainer.getScriptEngine()instanceof Compilable scriptCompiler) {
```

Comment:
```suggestion
                        && scriptEngineContainer.getScriptEngine() instanceof Compilable scriptCompiler) {
```

Reply:
This is the result of spotless, if I change it, `spotless:apply` removes the space again.

Reply:
Indeed, I was also just looking at the build failure and saw the same... Weird, but ok, let's leave it this way then.

Reply:
I'll check if this is fixed in newer versions of spotless, but that's another PR.